packages/giveaway/hardhat.config.ts

```diff
@@ -0,0 +1,21 @@
+import '@nomicfoundation/hardhat-toolbox';
+import '@dlsl/hardhat-markup';
+
```

Comment:
do you need to import solidity-coverage for coverage to work?

Reply:
@nomicfoundation/hardhat-toolbox takes care of that.